buildtools/maven/maven.go

```diff
@@ -97,7 +97,7 @@ func (m *Maven) DependencyTree(dir, project string) ([]Dependency, map[Dependenc
 	output, _, err := exec.Run(exec.Cmd{
 		Name: m.Cmd,
 		Dir:  dir,
-		Argv: []string{"dependency:tree", "--batch-mode", "--projects", project},
+		Argv: []string{"dependency:tree", "--batch-mode", "-Dscope=runtime"},
```

Comment:
I didn't see any use of putting it in the docs.

Let me search for it again 👍

Reply:
The behavior we want is:

 1. get the whole dep tree
 2. filter for the given project (or artifact) for dependency analysis
 3. filter for runtime scope only by default, but have this be configurable

Options 2/3 SHOULD be provided by the plugin, but [MSHARED-4](https://issues.apache.org/jira/projects/MSHARED/issues/MSHARED-4?filter=allopenissues) says that these don't function correctly due to the bug.

Reply:
Also the default might not be just runtime, we might also want compile as well.

Reply:
@xizhao 
1. Got it.
2. *-Dscope=runtime* seems to work even after *MSHARED-4*. That's weird, but the test shows that it works :/
3. Got it.


*runtime scope* gives both the *compile-time* as well as *run-time* dependencies. 
So that's fixed. 
Right?